### PR TITLE
CLI docs cleanup to remove legacy commands

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: 
- https://github.com/mattermost/mattermost/pull/17995
- https://github.com/mattermost/docs/issues/6412

Outstanding: Review redirects to ensure they're working as intended.

### CODE REVIEW

File: source/manage/command-line-tools.rst [hunk 1]
```diff
@@ -6,43 +6,30 @@ Command line tools
...
-
-**General administration**
+.. important::
 
--  Create teams
--  Create users
--  Assign roles to users
--  Reset user passwords
--  Invite users to teams
+  As of Mattermost v6.0, this CLI has been replaced with the `mmctl command line tool </manage/mmctl-command-line-tool.html>`__. However, `mattermost import </manage/command-line-tools.html#mattermost-import>`__ commands, `mattermost export </manage/command-line-tools.html#mattermost-export>`__ commands, and related subcommands, remain available and fully supported from Mattermost v6.0.
```
Comment (cwarnermm) [inline source/manage/command-line-tools.rst:11]: @sadohert @streamer45 - Please confirm whether Mattermost continues to support the following CLI parent/child commands: `mattermost import` & `mattermost export`. If these are supported, I'll ensure they're re-added. 

If they're no longer supported, please review the change scope of https://github.com/mattermost/mattermost/pull/17995 and let me know if other CLI commands need to remain documented.

File: source/manage/command-line-tools.rst [hunk 2]
```diff
@@ -111,2701 +103,341 @@ Options
...
-
-  For the *add*, *archive*, *delete*, *remove*, and *restore* commands, you can specify the *{channels}* value by {team}:{channel} using the team and channel URLs, or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search </manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
-    
-  For example, in the following URL the *{channels}* value is *myteam:mychannel*: ``https://example.com/myteam/channels/mychannel``
-
-  Also, the team and channel names in the URL should be written in lowercase.
-
-mattermost channel add
-~~~~~~~~~~~~~~~~~~~~~~
+  -  `mattermost db downgrade`_ - Roll back database migrations. Requires either an update plan to roll back to, or comma-separated version numbers to be rolled back.
```
Comment (cwarnermm) [inline source/manage/command-line-tools.rst:120]: @sadohert & @streamer45 - It's unclear to me whether Mattermost continues to support the `mattermost db downgrade` CLI command. Please confirm.

File: source/manage/command-line-tools.rst (hunk 1 above)
Comment (cwarnermm) [reply source/manage/command-line-tools.rst:11]: Please also let me know if there's a specific Mattermost release version we want to tie all CLI deprecations to specifically via docs, if not Mattermost v6.0.

File: source/manage/command-line-tools.rst (hunk 2 above)
Comment (streamer45) [reply source/manage/command-line-tools.rst:120]: I'll defer to @isacikgoz  as he'd know best.

Comment (isacikgoz) [reply source/manage/command-line-tools.rst:120]: It's actually freshly added command, so yes we will support it.

File: source/manage/command-line-tools.rst (hunk 1 above)
Comment (cwarnermm) [reply source/manage/command-line-tools.rst:11]: @isacikgoz - Can you help answer these outstanding questions?

Comment (nab-77) [reply source/manage/command-line-tools.rst:11]: import and export should be performed by `mmctl`

File: source/manage/command-line-tools.rst [hunk 3]
```diff
@@ -111,2701 +103,341 @@ Options
...
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
 
-mattermost channel remove
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Description
+  Commands for exporting data for compliance and for merging multiple Mattermost instances.
 
-.. note::
+Child Commands
```
Comment (nab-77) [inline source/manage/command-line-tools.rst:259]: Are these export options supported by `mmctl`? Seems an omission we should fix if not.

**Review by nab-77 — COMMENTED** [review]
More questions that answers. Sorry for that.

File: source/manage/command-line-tools.rst (hunk 1 above)
Comment (streamer45) [reply source/manage/command-line-tools.rst:11]: @cwarnermm 

They are still part of the server binary so unless we remove them there (breaking change) we should probably keep their documentation. 

As far as I know only the `import bulk` and `export bulk` commands have been ported to `mmctl` which is the recommended way of using that functionality so we should likely deprecate the `mattermost` binary equivalents.

/cc @nab-77

Comment (cwarnermm) [reply source/manage/command-line-tools.rst:11]: One more question about the ``mattermost import`` CLI command - one of it's documented child commands via ``--help`` is ``--slack``. Is this intended and desired?

Comment (streamer45) [reply source/manage/command-line-tools.rst:11]: That's another command that we should propbably deprecate/remove. As far as I understand it's been replaced by the dedicated [`mmetl`](https://github.com/mattermost/mmetl) tool which can be used in conjunction with `mmctl` to achieve the same functionality the legacy command provided.

Comment (cwarnermm) [reply source/manage/command-line-tools.rst:11]: Thank you so much for unblocking the remaining work needed for this docs update, @streamer45!

**Pull request merged**